servers/display_server.cpp

```diff
@@ -390,6 +390,9 @@ void DisplayServer::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("get_window_list"), &DisplayServer::get_window_list);
 	ClassDB::bind_method(D_METHOD("get_window_at_screen_position", "position"), &DisplayServer::get_window_at_screen_position);
 
+	ClassDB::bind_method(D_METHOD("set_allow_child_windows_in_fullscreen", "allow"), &DisplayServer::set_allow_child_windows_in_fullscreen);
+	ClassDB::bind_method(D_METHOD("get_allow_child_windows_in_fullscreen"), &DisplayServer::get_allow_child_windows_in_fullscreen);
```

Comment:
This could be a property I guess.

Reply:
That's gonna be the only property in the `DisplayServer`, maybe it better to add another window mode instead, like `WINDOW_MODE_FULLSCREEN` and `WINDOW_MODE_EXCLUSIVE_FULLSCREEN`?

Reply:
Changed it to `WINDOW_MODE_FULLSCREEN` (new behavior, with single pixel border) and `WINDOW_MODE_EXCLUSIVE_FULLSCREEN` (old behavior, with black flash on mode change, and no child windows).